doc/lua_api.txt

```diff
@@ -1391,6 +1391,19 @@ Displays distance to selected world position.
 * `offset`: offset in pixels from position.
 * `alignment`: The alignment of the waypoint.
 
+### `image_waypoint`
+
+Same as `image`, but does not accept a `position`; the position is instead determined by `world_pos`, the world position of the waypoint.
+
+* `scale`: The scale of the image, with 1 being the original texture size.
+  Only the X coordinate scale is used (positive values).
+  Negative values represent that percentage of the screen it
+  should take; e.g. `x=-100` means 100% (width).
+* `text`: The name of the texture that is displayed.
+* `alignment`: The alignment of the image.
+* `world_pos`: World position of the waypoint.
+* `offset`: offset in pixels from position.
+
```

Comment:
Correct me if I'm wrong, but I think this implemented in a different PR. Seems to have been accidentally added to this PR while rebasing.